eth/main.cpp

```diff
@@ -904,20 +904,19 @@ void interactiveMode(eth::Client* c, std::shared_ptr<eth::TrivialGasPricer> gasP
 		}
 		else if (c && cmd == "dumptrace")
```

Comment:
I have not tested those changes yet. Just make it compile in I think its correct.